### PR TITLE
refactor: Clean utils classes

### PR DESCRIPTION
**Pull request by conradoverta**
## Impact and Context
- Remove unused methods or arguments from ModelDBUtils
- Move things used by other components to CommonUtils

## Testing
- [ ] Unit test
- [ ] Deployed to dev env
- [ ] Other (explain) 

## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java [hunk 1]
```diff
@@ -78,6 +75,26 @@ public static boolean isEnvSet(String envVar) {
     return envVarVal != null && !envVarVal.isEmpty();
   }
 
+  public static void scheduleTask(
```
Comment (conradoverta) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:78]: used by registry

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java [hunk 2]
```diff
@@ -78,6 +75,26 @@ public static boolean isEnvSet(String envVar) {
...
+      TimerTask task, long initialDelay, long frequency, TimeUnit timeUnit) {
+    // scheduling the timer instance
+    ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
+    executor.scheduleAtFixedRate(task, initialDelay, frequency, timeUnit);
+  }
+
+  public static ModelDBException getInvalidFieldException(IllegalArgumentException ex) {
```
Comment (conradoverta) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:85]: used by registry

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java [hunk 3]
```diff
@@ -108,20 +125,15 @@ public static Object retryOrThrowException(
     throw ex;
   }
 
-  public static StatusRuntimeException logError(Throwable e) {
-    return logError(e, null);
-  }
-
-  public static <T extends GeneratedMessageV3> StatusRuntimeException logError(
```
Comment (conradoverta) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:115]: `defaultInstance` is not used

**Review by jkwatson-verta — APPROVED** [review]
👍🏼

**Pull request merged**